scripts/build/build_darwin_framework.py

```diff
@@ -71,12 +77,46 @@ def build_darwin_framework(args):
             "GCC_SYMBOLS_PRIVATE_EXTERN=NO",
         ]
 
-    if not args.ipv4:
+    options = {
+        'CHIP_INET_CONFIG_ENABLE_IPV4': args.ipv4,
+        'CHIP_IS_ASAN': args.asan,
+        'CHIP_IS_BLE': args.ble,
+        'CHIP_IS_CLANG': args.clang,
+    }
+    for option in options:
+        command += ["{}={}".format(option, "YES" if options[option] else "NO")]
+
+    # For now disable unguarded-availability-new warnings because we
+    # internally use APIs that we are annotating as only available on
+    # new enough versions.  Maybe we should change out deployment
+    # target versions instead?
+    cflags = ["${inherited}", "-Wno-unguarded-availability-new"]
+    ldflags = ["${inherited}"]
+
+    if args.clang:
         command += [
-            "CHIP_INET_CONFIG_ENABLE_IPV4=NO",
+            "CC={}".format(get_file_from_pigweed("clang++")),
```

Comment:
CC is the _c++_ compiler?  I would have thought we'd set CC to `clang` and CXX to `clang++` or something...

Reply:
Fixed.